src/ssd/ssd-extents.c

```diff
@@ -68,7 +69,8 @@ ssd_extents_update(struct ssd *ssd)
 	int height = view_effective_height(view, /* use_pending */ false);
 	int full_height = height + theme->border_width * 2 + ssd->titlebar.height;
 	int full_width = width + 2 * theme->border_width;
-	int extended_area = SSD_EXTENDED_AREA;
+	int border_width = MAX(0, MAX(rc.resize_minimum_area, theme->border_width));
```

Comment:
Why do we need the outer `MAX(0, ..)` here? if either of the vars are negative I'd assume other things will break as well.

Reply:
We don't, really, because `rc.resize_minimum_area` is already clipped to a non-negative value when parsing `rc.xml`. Removed.